code/clue.js

```diff
@@ -3,57 +3,274 @@
+  mrsPeacock,
+  colonelMustard,
+  mrsWhite,
+];
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+];
 
-const rooms = []
+const rooms = [
+  'Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Billiard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'Guest House',
+  'Patio',
+];
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
-const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
-}
+const randomSelector = (array) => {
+  return array[Math.floor(Math.random() * array.length)];
+};
 
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
+let mystery = {
+  killer: null,
+  weapon: null,
+  room: null,
+};
+//function to shuffle the suspects favourite weapon
+const randomFavouriteWeapon = () => randomSelector(weapons);
+
+const killerLoader = document.getElementById('load');
+const weaponLoader = document.getElementById('load2');
+const roomLoader = document.getElementById('load3');
+
+const killerHeadLine = document.getElementById('killer-headline');
+const weaponHeadLine = document.getElementById('weapon-headline');
+const roomHeadLine = document.getElementById('room-headline');
+
+//functions to show loaders and change the before the information
+const showKillerLoader = () => {
+  killerHeadLine.innerHTML = 'Loading killer-information..';
+  killerLoader.style.display = 'block';
+};
+const showWeaponLoader = () => {
+  weaponHeadLine.innerHTML = 'Loading weapon-information..';
+  weaponLoader.style.display = 'block';
+};
+const showRoomLoader = () => {
+  roomHeadLine.innerHTML = 'Loading room..';
+  roomLoader.style.display = 'block';
+};
+//function to show killer information
+const killerInformation = () => {
+  const killerCard = document.getElementById('killerCard');
+  const killerName = document.getElementById('killerName');
+  const killerImage = document.getElementById('killerImage');
+  const killerAge = document.getElementById('killerAge');
+  const killerOccupation = document.getElementById('killerOccupation');
+  const killerDescription = document.getElementById('killerDescription');
+  const killerFavouriteWeapon = document.getElementById(
+    'killerFavouriteWeapon'
+  );
+
+  mystery.killer = randomSelector(suspects);
+  killerLoader.style.display = 'none';
+  killerHeadLine.innerHTML = 'Killer-information:';
+  killerCard.style.background = mystery.killer.color;
+  killerName.innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`;
+  killerImage.src = mystery.killer.image;
+  killerAge.innerHTML = `${mystery.killer.age} years old`;
+  killerOccupation.innerHTML = `${mystery.killer.occupation}`;
```

Comment:
This could be simply
` killerOccupation.innerHTML = mystery.killer.occupation;`